etl/steps/data/garden/un/2023-10-09/plastic_waste.py

```diff
@@ -0,0 +1,151 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import owid.catalog.processing as pr
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+# Logger
+log = get_logger()
+
+REGIONS = [
+    "North America",
+    "South America",
+    "Europe",
+    "European Union (27)",
+    "Africa",
+    "Asia",
+    "Oceania",
+    "Low-income countries",
+    "Upper-middle-income countries",
+    "Lower-middle-income countries",
+    "High-income countries",
+    "World",
+]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("plastic_waste"))
```

Comment:
you could use `paths.load_dataset` instead. Then, you don't need to use `cast` or anything.

```suggestion
    ds_meadow = paths.load_dataset("plastic_waste")
```